Unfortunately I have been unable to get it to run in a project using `yarn link`. I end up with the following error:

`Module not found: You attempted to import ../esm/Slider/index.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.`

--- a/packages/material-ui-lab/src/Slider/Slider.js
+++ b/packages/material-ui-lab/src/Slider/Slider.js
@@ -539,7 +539,8 @@ const Slider = React.forwardRef(function Slider(props, ref) {
     touchId.current = undefined;
     document.body.removeEventListener('mousemove', handleTouchMove);
     document.body.removeEventListener('mouseup', handleTouchEnd);
-    document.body.removeEventListener('mouseenter', handleTouchEnd);
+    // eslint-disable-next-line no-use-before-define
+    document.body.removeEventListener('mouseenter', handleMouseEnter);
     document.body.removeEventListener('touchmove', handleTouchMove);
     document.body.removeEventListener('touchend', handleTouchEnd);
   });

